src/OgHelper.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\og\OgHelper.
+ */
+
+namespace Drupal\og;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Field\FieldException;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+
+/**
+ * OG helper methods.
+ */
+class OgHelper {
+
+  /**
+   * Return TRUE if a field can be used and has not reached maximum values.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The content entity to check the field cardinality for.
+   * @param string $field_name
+   *   The field name to check the cardinality of.
+   *
+   * @return bool
+   */
+  public static function checkFieldCardinality(ContentEntityInterface $entity, $field_name) {
```

Comment:
I think it depends on the usage tbh. Some function you will want to be able to call on more entities, some do not need that so much, like this one. We need to type hint `ContentEntityInterface` or `FieldableEntityInterface` (which might be a better choice) here as you have no methods available for getting field definitions etc..